package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "forever-style-guide",
+  "version": "1.0.0",
```

Comment:
The current version defined in version.rb is 3.0.35, not sure if that has any play here at this point, but probably better to be consistent?
Or do you want to mark this is as moving towards a v1 for web app implementation?